src/fonts/OpenDyslexic/OpenDyslexic/OpenDyslexic-Bold.svg

```diff
@@ -0,0 +1,244 @@
+<?xml version="1.0" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd" >
```

Comment:
Oh, that's interesting. I'll remove these.


On 7/14/2016 1:48 AM, Daniel Weck wrote:
>
> In src/fonts/OpenDyslexic/OpenDyslexic/OpenDyslexic-Bold.svg 
> <https://github.com/readium/readium-js-viewer/pull/540#discussion_r70762812>:
>
> > @@ -0,0 +1,244 @@
> > +<?xml version="1.0" standalone="no"?>
> > +<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd" >
>
> Hello @derekriemer <https://github.com/derekriemer> in this commit 
> there are duplicated font files (inside a new OpenDyslexic subfolder) 
> in addition to src/js/ReaderSettingsDialog.js
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/readium/readium-js-viewer/pull/540/files/41229f6bade6f63f04851ed68233d9b5a76478b3..642bf5d9a886bdb2e765d61e05ee366db8b5a0f4#r70762812>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AFGivWy9_e6ugJhPTNH8pnWAXFR9rY5Jks5qVem4gaJpZM4I8jm9>.
>

-- 
------------------------------------------------------------------------


    Derek Riemer

  * Department of computer science, third year undergraduate student.
  * Proud user of the NVDA screen reader.
  * Open source enthusiast.
  * Member of Bridge Cu
  * Avid skiier.

Websites:
Honors portfolio <http://derekriemer.com>
Awesome little hand built weather app! 
<http://django.derekriemer.com/weather/>

email me at derek.riemer@colorado.edu <mailto:derek.riemer@colorado.edu>
Phone: (303) 906-2194